src/feature/dircache/dircache.c

```diff
@@ -166,22 +166,16 @@ write_http_response_header_impl(dir_connection_t *conn, ssize_t length,
   buf_free(buf);
 }
 
-/** As write_http_response_header_impl, but sets encoding and content-typed
- * based on whether the response will be <b>compressed</b> or not. */
+/** As write_http_response_header_impl, but translates compress_method_t into
```

Comment:
The argument name also changes from `method to `encoding`.
So let's say:
```suggestion
/** As write_http_response_header_impl, but translates method into
```

You can keep the types as well if you want.